uint8 REQUEST_CONTROL=0
uint8 ARM=1
uint8 DISARM=2

uint8 command
---
bool success